xcode/Subconscious/Shared/Components/Common/Forms/ValidatedTextField.swift

```diff
@@ -12,14 +12,20 @@ struct ValidatedTextField: View {
     var placeholder: String
     @Binding var text: String
     var caption: String
-    var isValid: Bool = true
+    var hasError: Bool = false
```

Comment:
This allows for clearer of expression between the different layers of concern. imo a text field should not show an error if it has not been touched by the user yet, even if the underlying value is invalid. It's incorrect to say that the field is valid before it's been touched, technically it's invalid if we tried to submit the form, but it also should not have a visible error.

Flipping the logic keeps the ValidatedTextField dumb and lets FormField expose both isValid and hasError properties and drive the specific expected behaviour.